packages/jest-types/src/Config.ts

```diff
@@ -468,6 +468,7 @@ export type Argv = Arguments<
     roots: Array<string>;
     runInBand: boolean;
     selectProjects: Array<string>;
+    ignoreProjects: Array<string>;
```

Comment:
Should follow alphabetical order, perhaps?

Reply:
nice catch! thanks.